src/js/manager.js

```diff
@@ -29,17 +34,26 @@ $(function () {
     }
   });
 });
+
 /*
  * function for change option for command selection
  */
-function changeOption() {
+function changeCommand() {
   var option = $('#option');
   option.empty();
   var command = $('#command').val();
   for (var i = 0; i < options[command].length; i++) {
     option.append(new Option('-' + options[command][i], options[command][i]));
   }
+  autocomplete();
+  showKeywordBox();
 }
+
+function changeOption(){
+  autocomplete();
```

Comment:
In my opinion, following flow is better for user perspective. if user change the option then
1. clear command text box.
2. trigger command text change event.
What do you think?